upgrade-notes/backend_20240703_131550.md

```diff
@@ -0,0 +1,13 @@
+#### administration optimization ([#{pullRequestId}](https://github.com/shopsys/shopsys/pull/{pullRequestId}))
+
+-   `Shopsys\FrameworkBundle\Component\Paginator\QueryPaginator` class was changed:
+    -   `getResult()` method changed its interface:
+        ```diff
+            public function getResult(
+                int $page = 1,
+                ?int $pageSize = null,
+        +       ?int $totalCount = null,
+            ): PaginationResult {
+            ```
+-   many strict types have been added so run `composer install`, `php phing standards-fix` and `php phing phpstan` commands, which will probably fail on errors caused by incompatibility strict types, if so fix them manually
```

Comment:
can you please write this sentence similarly to the moved feature?

Reply:
I have added some introduction for it